### PR TITLE
clang-cl support for native Windows builds.

### PR DESCRIPTION
**Pull request by dave-juicelabs**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -64,7 +64,7 @@ if not enable_trace
   add_project_arguments('-DVKD3D_NO_TRACE_MESSAGES', language : 'c')
 endif
 
-vkd3d_external_includes = [ './subprojects/Vulkan-Headers/include', './subprojects/SPIRV-Headers/include' ]
+vkd3d_external_includes = [ './subprojects/SPIRV-Headers/include' ]
```
Comment (Joshua-Ashton) [inline meson.build:67]: Looks like an unintentional change here.

If not, why were the header includes removed? We really don't want a dependency on whatever outdated crap the SDK is shipping.

**Review by Joshua-Ashton — CHANGES_REQUESTED** [review]

File: meson.build (hunk 1 above)
Comment (dave-juicelabs) [reply meson.build:67]: It was unintentional to include that. I will fix this and also update the commit message as requested in the other PR.

**Review by Joshua-Ashton — CHANGES_REQUESTED** [review]
Miiinor nit, commit prefix should be `build:`

**Review by Joshua-Ashton — APPROVED** [review]
LGTM!

**Pull request merged**